lib/matplotlib/spines.py

```diff
@@ -265,11 +265,17 @@ def _adjust_location(self):
                 self._path = mpath.Path.arc(np.rad2deg(low), np.rad2deg(high))
 
                 if self.spine_type == 'bottom':
-                    rmin, rmax = self.axes.viewLim.intervaly
+                    if self.axis is None:
```

Comment:
Added this check in case someone somewhere is using an arc spine that has not registered an axis.  I'm not sure how likely that is though, and I did not find anything with [this search](https://github.com/search?q=arc_spine+language%3APython+NOT+is%3Afork+NOT+path%3A**%2Fmatplotlib%2F**+NOT+path%3A**%2Fsite-packages%2F**&type=code).

Reply:
I'd say better safe than sorry, but is it possible to create a test case which triggers this? (Not saying that there should be, but maybe no harm to add one, especially if it doesn't require an image test.)

Reply:
I added a slightly contrived smoke test.